src/index.ts

```diff
@@ -91,9 +91,24 @@ declare module "http" {
   }
 }
 
+type RequestType =
+  | { headers: IncomingHttpHeaders }
+  | { cookies: { [key: string]: string } };
```

Comment:
So in middleware we can access req.cookies directly?

Reply:
To support not only Next.js Middleware but also Deno and Cloudflare Workers we should not assume this is available. But use the format of the Web API Request / Response, which will be compatible with Next.js Middleware.

Reply:
(we did not assume that in an update to this file: https://github.com/vvo/iron-session/pull/510/commits/b9fc40839313ebe282b0dd1f7f621a403bfe07be, more to come!)